resources/servo.css

```diff
@@ -124,19 +124,17 @@ img {
   overflow-clip-margin: 0 !important;
 }
 
-input:not([type=radio i]):not([type=checkbox i]):not([type=reset i]):not([type=button i]):not([type=submit i]),
-textarea {
+input:not([type=radio i]):not([type=checkbox i]):not([type=reset i]):not([type=button i]):not([type=submit i]) {
   cursor: text;
   overflow: hidden !important;
-  -servo-overflow-clip-box: content-box;
-}
-
-input:not([type=radio i]):not([type=checkbox i]):not([type=reset i]):not([type=button i]):not([type=submit i]) {
+  overflow-clip-margin: content-box;
```

Comment:
Actually this isn't working yet because currently we only support `<length>` values, but I guess you can leave it for the future.